Library/Homebrew/utils/pypi.rb

```diff
@@ -327,12 +330,16 @@ def self.update_python_resources!(formula, version: nil, package_name: nil, extr
     # Resolve the dependency tree of all input packages
     show_info = !print_only && !silent
     ohai "Retrieving PyPI dependencies for \"#{input_packages.join(" ")}\"..." if show_info
-    found_packages = pip_report(input_packages, python_name:, print_stderr: verbose && show_info)
+    print_stderr = !(verbose && show_info).nil?
```

Comment:
```suggestion
    print_stderr = if verbose && show_info
      true
    else
      false
    end
```
is a bit easier to read